src/vea/vea_hint.c

```diff
@@ -54,12 +54,14 @@ hint_cancel(struct vea_hint_context *hint, uint64_t off, uint64_t seq_min,
 		 */
 		hint->vhc_off = off;
 		return 0;
-	} else if (hint->vhc_seq > seq_max) {
+	} else if (hint->vhc_seq >= seq_max) {
```

Comment:
how could "==" happen?

Reply:
Let's say we reserve a A, B, C extent.

Then we just cancel A, C, in this case,
hint->vhc_seq = seq_max, and is_rsrv_interleaved is true?